build.gradle

```diff
@@ -29,6 +29,13 @@ deploy {
                 // getTargetTypeClass is a shortcut to get the class type using a string
 
                 frcJava(getArtifactTypeClass('FRCJavaArtifact')) {
+                    // Enable VisualVM connection
```

Comment:
We don't yet have a way to enable/disable this, and it likely takes some network bandwidth as well as CPU cycles.

Let's not commit it to main, and add it to build.gradle when needed.

Reply:
Can't we use compile-time parameters to add these? Is that difficult to do with VSCode? Or maybe we need an additional build target?